envi/archs/i386/disasm.py

```diff
@@ -1038,7 +1038,7 @@ def disasm(self, bytez, offset, va):
     def ameth_0(self, operflags, operval, tsize, prefixes):
         # Special address method for opcodes with embedded operands
         if operflags & opcode86.OP_REG:
-            if prefixes & PREFIX_OP_SIZE:
+            if prefixes & PREFIX_OP_SIZE and operval & RMETA_NMASK == operval:
```

Comment:
No to the first, yes to the second. 
From the intel manual (Section 2.1.1):
```
The operand-size override prefix allows a program to switch between 16- and 32-bit operand sizes. Either size can
be the default; use of the prefix selects the non-default size.
```
For now this will work. I've already got a note for myself to come back to the handling of both 66 and 67 prefixes, since better handling of 66 requires plumbing through the modesize further into the disassembler, which is a riskier task that is a bit safer to put into its own PR.